src/gcp/secretManager.ts

```diff
@@ -1,5 +1,22 @@
+import * as iam from "./iam";
+
 import { logLabeledSuccess } from "../utils";
-import * as api from "../api";
+import { FirebaseError } from "../error";
+import { Client } from "../apiv2";
+import { secretManagerOrigin } from "../api";
+
+// Matches projects/{PROJECT}/secrets/{SECRET}
+const SECRET_NAME_REGEX = new RegExp(
+  "projects\\/" +
+    "((?:[0-9]+)|(?:[A-Za-z]+[A-Za-z\\d-]*[A-Za-z\\d]?))\\/" +
```

Comment:
nit: it's weird to see you combine "[0-9]" and "[\d]" in the same capture group

Reply:
Also, are we sure we want to do project #3509

Reply:
This regex is used to parse Secret Manager output, so we do need project (I'm not sure how the linked issue relates to this?)